packages/arb-token-bridge-ui/src/hooks/arbTokenBridge.types.ts

```diff
@@ -95,6 +95,7 @@ export type L2ToL1EventResultPlus = L2ToL1EventResult & {
 
 export type WithdrawalInitiated = EventArgs<WithdrawalInitiatedEvent> & {
   txHash: string
+  timestamp?: BigNumber
```

Comment:
Used to sort sent and received txs for event logs as they are displayed together now